probatus/utils/__init__.py

```diff
@@ -30,4 +30,5 @@
 __all__ = ['NotFittedError', 'DimensionalityError', 'UnsupportedModelError', 'Scorer', 'assure_numpy_array', 'assure_pandas_df', 'check_1d',
            'ApproximationWarning', 'class_name_from_object', 'get_scorers', 'assure_list_of_strings',
            'assure_list_values_allowed', 'warn_if_missing','TreePathFinder', 'check_numeric_dtypes',
-           'plot_distributions_of_feature', 'assure_column_names_consistency', 'shap_calc', 'shap_to_df', 'calculate_shap_importance']
+           'plot_distributions_of_feature', 'assure_column_names_consistency', 'shap_calc', 'shap_to_df', 'calculate_shap_importance',
```

Comment:
Minor detail, but just wondering if you ready want to expose all internal functions to users?

Reply:
I think they are already exposed that they can import them, the assumption is if they are not described in the API they will not be used so much for now. In the future we can describe them in the docs.